package-dev/Runtime/SentryInitialization.cs

```diff
@@ -140,13 +143,17 @@ private Il2CppMethods _il2CppMethods
         private static void Il2CppNativeStackTraceShim(IntPtr exc, out IntPtr addresses, out int numFrames, out string? imageUUID, out string? imageName)
         {
             imageName = null;
-            il2cpp_native_stack_trace(exc, out addresses, out numFrames, out imageUUID);
+            // Unity 2020 does not *return* a newly allocated string as out-parameter,
+            // but rather expects a pre-allocated buffer it writes into.
+            char[] uuidBuffer = new char[32 + 1];
```

Comment:
Where is the `32+1` coming from?

Reply:
its the uuid + terminating nul. I added a comment to explain.